jcl/jpp_configuration.xml

```diff
@@ -243,6 +243,33 @@
 		<parameter name="msg:outputdir" value="java.base/share/classes/com/ibm/oti/util"/>
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
+
+	<configuration
+		  label="OPENJ9-RAWBUILD"
```

Comment:
This should include a version, such as OPENJ9-RAWBUILD11. Similarly the flag. Another naming suggestion: JAVA11-RAW.

Reply:
This is really first time that a raw build need its own flag to workaround a `JCL` path not available to raw builds. I don't expect there are many more similar cases in the future. In additional raw builds intend to discover issues at earlier stage before normal builds can be performed. When a `JCL` level has a routine build, there is no need to perform raw builds for that level any more such as `Java 9`.
In worst scenario there is a need for another raw build `pConfig`, this generic raw build flag can be renamed first then add another one later.